src/param.c

```diff
@@ -33,6 +33,10 @@ static void init_param_float(param_id_t id, char name[PARAMS_NAME_LENGTH], float
 // function definitions
 void init_params(void)
 {
+  for(uint8_t i = 0; i < PARAMS_COUNT; i++)
+  {
+    init_param_int(i, "DEFAULT", 0);
+  }
```

Comment:
I used to have something like this here because we weren't initializing every param before, but once that was fixed I got rid of it because it seems like a lot of unnecessary work. Is there a specific reason you put it back in?